setting_up_firely_server/configuration/mongodb/database_security.rst

```diff
@@ -0,0 +1,42 @@
+Database security
+=================
+
+The following explores some of the various security measures that exist in MongoDB and their applicability to a Firely Server MongoDB database.
+
+Access Control
+--------------
+Access control should be enabled on the MongdDB environment. With this enabled, you will have to make an application user with the appropriate role and use this user to connect Firely Server to the MongoDB environment.
+This prevents unauthorized access to the database.
+
+`More information <https://www.mongodb.com/docs/manual/tutorial/enable-authentication/>`_
```

Comment:
Can we give any details on the rights that should minimally be assigned to this user? 
And the rights that are minimally required for a user to use for AutoUpgrade (as you can set a different connection string for that)?